locales/en/index.yml

```
@@ -894,8 +894,8 @@ wallet:
     noName: Name unavailable
     loading: Loading bank list...
   accessibility:
-    sectionCardLabel: "Update payment methods"
-    sectionCardHint: "Update the list of payment methods"
+    sectionCardLabel: "Payment methods"
+    sectionCardHint: "double tap to update the list"
```

```suggestion
    sectionCardLabel: "Update payment methods"
    sectionCardHint: "Update the list of payment methods"
```

During the story https://pagopa.atlassian.net/browse/IA-47 we decided to change the text with "Update payment methods / Update the list of payment methods" in order to have a text that explains what the button do and additionaly give some other feedback in the hint, without giving explicit references to specific implicit actions, such as double taps.
Maybe that the story IA-51 it hasn't been updated after these changes, but it should reflect the same behaviour, what you think?

---

yes looks good!